frappe/desk/doctype/system_console/system_console.js

```diff
@@ -79,14 +79,28 @@ frappe.ui.form.on('System Console', {
 		let timestamp = new Date();
 		frappe.call('frappe.desk.doctype.system_console.system_console.show_processlist').then(r => {
 			let rows = '';
-			for (let row of r.message) {
-				rows += `<tr>
-					<td>${row.Id}</td>
-					<td>${row.Time}</td>
-					<td>${row.State}</td>
-					<td>${row.Info}</td>
-					<td>${row.Progress}</td>
-				</tr>`
+			if(r.message['db_type'] == 'postgres'){
+				for (let row of r.message['data']) {
+					rows += `<tr>
+						<td>${row.pid}</td>
+						<td>${row.query_start}</td>
+						<td>${row.state}</td>
+						<td>${row.query}</td>
+						<td>${row.wait_event}</td>
```

Comment:
`wait_event` is not the equivalent of `Progress` in mariadb ([ref](https://mariadb.com/kb/en/show-processlist/#description)) and there is no other similar alternative in the docs ([ref](https://www.postgresql.org/docs/9.6/monitoring-stats.html#PG-STAT-ACTIVITY-VIEW)). so maybe it's okay to leave as is?

OR

Maybe we can also change the name/placeholder to represent the change?
https://github.com/frappe/frappe/blob/d4b23185690d03249e079489710f676bde817f50/frappe/desk/doctype/system_console/system_console.js#L99
This will be a bit overkill tho 😅

Reply:
wait event is useful to track the query status. We can change the title instead?

Reply:
Yep that's what I meant by name/placeholder ..sorry didn't know it was called title 😅

Reply:
what should I rename it to?

Reply:
`Progress/Wait Event` is what I was thinking since the title is same as the queried column but am not completely sure.

@gavingdsouza any suggestions?

Reply:
also, was thinking why we don't have `db_type` passed to the UI on boot? people will not have to do things like [this](https://github.com/frappe/frappe/pull/16212/files#diff-1908468e81fc7f28820970ab43bede22bf1052ba48aa6f86006d6328ce648b50R50) then. 
is this due to any security thing?

Reply:
also, was thinking why we don't have `db_type` passed to the UI on boot? people will not have to do things like [this](https://github.com/frappe/frappe/pull/16212/files#diff-1908468e81fc7f28820970ab43bede22bf1052ba48aa6f86006d6328ce648b50R50) then. 
is this due to any security thing?

Reply:
No need to add `db_type` in boot info we can handle this from backend itself

Reply:
yep but was thinking why we don't expose it 🤔  ..maybe in the future we choose to have some postgres/mariadb specific feature it might be helpful in that case i think

Reply:
In future if we build postgres/mariadb specific feature we can add this, but currently when we can handle this in backend then why to add it?

Reply:
ah this was more of a thought rather than something which had to be changed in this pr (though the thought did arise from seeing that commit 😅 ) ..sorry for the misunderstanding.